Riot/ViewController/ReadReceiptsViewController.m

```diff
@@ -0,0 +1,161 @@
+/*
+ Copyright 2017 Aram Sargsyan
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "ReadReceiptsViewController.h"
+#import <MatrixKit/MatrixKit.h>
+
+@interface ReadReceiptsViewController () <UITableViewDataSource, UITableViewDelegate>
+
+@property (nonatomic) MXRestClient* restClient;
+@property (nonatomic) MXSession *session;
+
+@property (nonatomic) NSArray <MXRoomMember *> *roomMembers;
+@property (nonatomic) NSArray <UIImage *> *placeholders;
+@property (nonatomic) NSArray <MXReceiptData *> *receipts;
+
+@property (weak, nonatomic) IBOutlet UIView *overlayView;
+@property (weak, nonatomic) IBOutlet UILabel *titleLabel;
+@property (weak, nonatomic) IBOutlet UIView *containerView;
+@property (weak, nonatomic) IBOutlet UITableView *receiptsTableView;
+
+@end
+
+@implementation ReadReceiptsViewController
+
+#pragma mark - Public
+
++ (void)openInViewController:(UIViewController *)viewController withRestClient:(MXRestClient *)restClient session:(MXSession *)session withRoomMembers:(NSArray <MXRoomMember *> *)roomMembers placeholders:(NSArray <UIImage *> *)placeholders receipts:(NSArray <MXReceiptData *> *)receipts
+{
+    ReadReceiptsViewController *receiptsController = [[[self class] alloc] initWithNibName:NSStringFromClass([self class]) bundle:nil];
+    receiptsController.restClient = restClient;
+    receiptsController.session = session;
+    
+    receiptsController.roomMembers = roomMembers;
+    receiptsController.placeholders = placeholders;
+    receiptsController.receipts = receipts;
+    
+    receiptsController.providesPresentationContextTransitionStyle = YES;
+    receiptsController.definesPresentationContext = YES;
+    receiptsController.modalPresentationStyle = UIModalPresentationOverFullScreen;
+    receiptsController.modalTransitionStyle = UIModalTransitionStyleCrossDissolve;
+    
+    [viewController presentViewController:receiptsController animated:YES completion:nil];
+}
+
+#pragma mark - Lifecycle
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    [self configureViews];
+    [self configureReceiptsTableView];
+    [self addOverlayViewGesture];
+}
+
+- (void)didReceiveMemoryWarning
+{
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Views
+
+- (void)configureViews
+{
+    self.containerView.layer.cornerRadius = 20;
+    self.titleLabel.text = @"Read Receipts List";
```

Comment:
Define a new key-value in Vector.strings to handle translation of this title.
Then use the macro: 
`NSLocalizedStringFromTable(@"xxx", @"Vector", nil)`, where xxx is your key.